recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - flex            # [unix]
     - winflexbison    # [win]
   host:
-    - boost-cpp
+    - libboost-devel
```

Comment:
Ha, starting off with a corner-case right away. For https://github.com/regro/cf-scripts/pull/1668, it wasn't 100% clear in all cases that a multi-output recipe might not end up relying on the libs by having a global boost host-dep and then run-deps in the outputs; hence the conservative choice of transforming this to the full library here. In this case we don't need it though, I'll fix that.